src/seedu/addressbook/data/person/Unit.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's unit in the address book.
+ */
+public class Unit {
+	
+	private String unit;
+	
+	public Unit(String unit_Num){
```

Comment:
Java convention is camel notation. To avoid confusion between class variables and arguments you can add an _ as a prefix to the argument name. i.e. _unitNum.